### PR TITLE
Support for JSON type columns

### PR DESCRIPTION
**Pull request by xande0812**
Thank you for the great tool.

In my project, I am using PostgreSQL's JSONB column. I would love to use this tool for testing, but it doesn't handle JSONB columns well. So I made some modifications on my own. What do you think?

I feel like there is some repetition in the code for supporting JSONB and sqlite's JSON support. Do you have any good ideas? Or do you think it's fine as it is?

### CODE REVIEW

File: db.go [hunk 1]
```diff
@@ -157,13 +158,32 @@ func (rnr *dbRunner) Run(ctx context.Context, q *dbQuery) error {
 								return fmt.Errorf("invalid column: evaluated %s, but got %s(%v): %w", c, t, s, err)
 							}
 							row[c] = d
+						case t == "JSONB": // PostgreSQL JSONB
```
Comment (k1LoW) [inline db.go:161]: > I feel like there is some repetition in the code for supporting JSONB and sqlite's JSON support. Do you have any good ideas? Or do you think it's fine as it is?

I'm fine with the implementation as it is 👍

**Review by k1LoW — APPROVED** [review]
LGTM! Thank you!!!

**Review by k1LoW — APPROVED** [review]
LGTM! Thank you!!!

**Comment by xande0812** [comment]
Thank you for merging!

**Pull request merged**